### ISSUE TITLE
Why is there a temperature range when setting AC in automatic?

### ISSUE THREAD

**Issue by ghost** [teodoradima/homebridge-samsung-air-conditioner-smart-things#11]
**Describe Your Problem:**

I don't understand why when I set the AC to automatic in HomeKit, it shows a temperature range selector that says "keep the temperature between xx ° C and yy ° C". Is this supposed to happen?

I don't know if it is a bug or it is expected to be so, but in any case there is something in this selector that doesn't work: the air conditioner does not correctly receive the command and does not understand the range of temperatures that is given.
Even if I try to set the selector as I wish (eg "keep the temperature between 21 °C and 24 °C") the AC does not accept both values, but only one of the two.
And I just need to exit from HomeKit and re-enter, to notice that the AC forgets the range set and places both ends of the slider on a single temperature.
This makes the selector difficult to manage and unstable in behavior.

Is it possible to make the selector set a single temperature in automatic mode (as happens for hot and cold)?

**Plugin Config:**

```json
{
    "bridge": {
        "name": "Homebridge 5199",
        "username": "0E:16:CF:8E:E6:59",
        "port": 51947,
        "pin": "650-32-218"
    },
    "accessories": [],
    "platforms": [
        {
            "name": "Config",
            "port": 8581,
            "platform": "config"
        },
        {
            "devices": [
                {
                    "name": "TV",
                    "ip": "192.168.1.104",
                    "mac": "2c:2b:f9:90:8c:6c",
                    "pollingInterval": 5,
                    "hideTvService": false,
                    "volumeLimit": 100,
                    "volumeControl": "none",
                    "channelControl": false,
                    "ccRemoteRemap": {
                        "information": "HOME"
                    }
                }
            ],
            "platform": "webostv"
        },
        {
            "name": "homebridge-samsung-air-conditioner-smart-things",
            "token": "540d5d01-69b1-4ccd-8ce8-bd6c80cc67a6",
            "temperatureUnit": "C",
            "showHumidity": false,
            "windFreeSupported": false,
            "platform": "HomebridgeSamsungACSmartThings"
        }
    ]
}
```

**Screenshots:**
![IMG_2284](https://user-images.githubusercontent.com/24357017/174473221-c627b932-4add-4adf-aca6-151ecdac1b16.jpg)

Video: https://drive.google.com/file/d/1tZe5dzaB4S6kQjGzByYj5hx-E5rQKYZc/view?usp=sharing

**Environment:**

* **Plugin Version**: 1.0.9
* **Homebridge Version**: 1.4.1
* **Node.js Version**: 16.13.1
* **NPM Version**: 8.3.0
* **Operating System**: Raspbian GNU/Linux Bullseye (11)

**Comment by EasyBizz** [comment]
Agreed, this seems useless. The setting only gets confusing and often doesnt work. Can we edit this somewhere in the files?
